ros2topic/ros2topic/api/__init__.py

```diff
@@ -253,3 +256,78 @@ def add_qos_arguments(parser: ArgumentParser, subscribe_or_publish: str, default
         help=(
             f'Quality of service liveliness lease duration setting to {subscribe_or_publish} '
             'with (overrides liveliness lease duration value of --qos-profile option'))
+
+def extract_qos_arguments(args):
```

Comment:
It's definitely useless. I had a few alternatives in mind but none that I was satisfied with:
- just pass `args` to the `_rostopic_verb` functions as the `qos` parameter. But in that case I might as well remove the other arguments from `_rostopic_verb` since `args` has all of them
- split each qos argument into their own function parameters but that would have added 6 parameters to both `_rostopic_verb` and `choose_qos`
I'm not sure what would best fit the style of this package. What do you think ?